specification/validated-relationships-service-api.yaml

```diff
@@ -481,36 +753,215 @@ components:
                           properties:
                             valueString:
                               type: string
-                              example: "LS1 4AP"
+                              example: "Child is too young"
 
-    QuestionnaireItem_RequestedServices:
+    QuestionnaireItem_Patient:
       type: object
       properties:
         linkId:
           type: string
-          enum: ["requested_services"]
+          enum: ["patient"]
         text:
           type: string
```

Comment:
Suggest adding an example "Details about the patient" for parity with most other fields, currently this one comes out as `"text": "string"` in the generated swagger docs.